flinkprocessor/build.gradle

```diff
@@ -17,7 +17,7 @@ apply plugin: "application"
 apply plugin: "maven-publish"
 apply plugin: "com.github.johnrengelman.shadow"
 
-group = "io.pravega.example.video"
+group = "io.pravega.examples.video"
```

Comment:
Why did this change? Were corresponding Helm charts updated to use this new name?

Reply:
fixed it